recipes/zlib/all/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
It's not, due to the cmake_policy requirement, which was added in 3.15 (according to a quick Google search)

Reply:
I know but `conan_basic_setup()` should take care of vc runtime regardless of CMP0091 policy. If you bump to 3.15, you don't need to force CMP0091 to NEW anyway because it would be the default behavior.

Reply:
Well, maybe it's a conan issue. @memsharded @lasote @czoido. Is new msvc compiler type compatible with old CMake helper + conan_basic_setup()?

Reply:
> is new msvc compiler type compatible with old CMake helper + conan_basic_setup()? 

I would say it was fixed at Conan 1.43.2 (https://github.com/conan-io/conan/pull/10195/files) At this time we are running 1.43.1 here. (soon will be an upgrade to 1.43.3 with some backports from 1.44.1)